Makefile

```diff
@@ -147,4 +147,5 @@ $(C_BUILDDIR)/%.o: $(C_SUBDIR)/%.c $$(c_dep)
 	$(AS) $(ASFLAGS) -o $@ $(C_BUILDDIR)/$*.s
 
 rev1:         ; @$(MAKE) GAME_REVISION=1
-compare_rev1: ; @$(MAKE) GAME_REVISION=1 COMPARE=1
\ No newline at end of file
+compare_rev1: ; @$(MAKE) GAME_REVISION=1 COMPARE=1
+clean_rev1:   ; @$(MAKE) GAME_REVISION=1 clean
```

Comment:
this is the most crude way to do this i'll need to figure out something better later